docs/extend/modular-resources/configure.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Configure your module and modular resource"
+linkTitle: "Configure"
+weight: 30
+type: "docs"
+tags: ["server", "rdk", "extending viam", "modular resources", "components", "services"]
+description: "Use the Viam module system to implement custom resources that can be included in any Viam-powered robot."
+no_list: true
+---
+
+Configure a module on your robot to make one or more modular resources available for configuration.
+
+If you want to create your own modular resource, you must first follow [these steps](/extend/modular-resources/create/) to code your own module and generate an executable.
+If you are using a pre-built modular resource, make sure you install the module and determine the filename of [the module's executable](/extend/modular-resources/create/#compile-the-module-into-an-executable).
+
+Follow these steps to configure the module and modular resource:
```

Comment:
Shouldn't we say resources?  Since multiple can be provided by a single module?

Reply:
I think that this is saying "hey you are adding a modular resource here's how you do that one task," rather than saying "here's how modular resources are configured in general," so I think the singular works well